packages/gafl-webapp-service/src/routes/__tests__/back-links.spec.js

```diff
@@ -109,6 +110,16 @@ describe('The address-entry page', () => {
   })
 })
 
+describe('The licence-fulfilment page', () => {
+  const n = journeyDefinition.find(n => n.current.page === LICENCE_FULFILMENT.page)
```

Comment:
Please could you move this inside each test? Feels a bit dirty asserting on the same instance for each test.